wasm-fuzzer/wafl.sh

```diff
@@ -0,0 +1,83 @@
+currentID=$$
+
+echo "wafl ID $currentID"
+
+export DOCKER_INTERFACE_SRC=$(pwd)/wafl
+export DOCKER_AFL_INPUT=$DOCKER_INTERFACE_SRC/in-$currentID
+
+export SWAM_SOCKET_PORT=9999
+export SWAM_SOCKET_HOST=0.0.0.0
+export DOCKER_AFL_OUTPUT=out-$currentID
+
+export REQUIRED_BYTES=4
+
+
+export DOCKER_LOGS=$DOCKER_INTERFACE_SRC/out
+
+export DUMMY_TESTING_AFL=False
+
+
+export AFL="$(pwd)/aflpp/afl-fuzz"
+export LOCAL_AFL="True"
+export WASI_FILTER="True"
+export WASI=True
+export WITH_TIMEOUT="False"
+export INFER_INIT_VALUES="True"
+export INIT_VALUES_SEED="0"
+export TIMEOUT=60 # 60 seconds
+
+mkdir -p $DOCKER_AFL_INPUT
+
+## Replace env variables
+set -a
+source .env
+set +x
+
+if [[ $1 == *.wat ]]; then WAT_ARG="--wat"; fi
+if [[ $WASI == "True" ]]; then WASI_ARG="--wasi"; fi
+if [[ $WASI_FILTER == "True" ]]; then WASI_ARG="$WASI_ARG -r"; fi
+if [[ $WITH_TIMEOUT == "True" ]]; then TIMEOUT="timeout --foreground $TIMEOUT"; else TIMEOUT=""; fi
+
+echo "Infering signature for wasm"
+
+echo "java -jar fuzzing-server-swam/out/cli/assembly/dest/out.jar infer $WAT_ARG $1 $2"
```

Comment:
I think this should be moved into entrypoint_afl.sh. Then we can remove the argument passing into `out.jar run_server` for SWAM and just call the function again directly in Scala when the server starts up.

Reply:
And we can remove all of this code: `ALL_ARG_TYPES="$ALL_ARG_TYPES --argType $element";` a couple lines down.